### DISCUSSION TITLE
Is there a way to specify worker resources per task in the custom graph interface?

### DISCUSSION THREAD

**Discussion by dask-user** [dask/distributed#7391] [Q&A]
Hi, I see that `client.get` allows me to specify `resources` for a custom graph execution. However, as far as I can tell, this applies the resources to all tasks in the custom graph. Is there a way to specify the desired resources per task?

Something like this:

```
client.get(
    {
        'taskA': (...),
        'taskB': (...),
        'taskC': (...),
        'taskD': (...),
    },
    resources={
        'taskA': {'MEMORY': 5000, 'CPU': 2},
        'taskD': {'CPU': 2}
    }
)
```

**Comment by ncclementi** [comment]
@dask-user as the discussions section mentions, you would have a better chance of getting an answer by posting on the dask discourse
https://dask.discourse.group/
